### ISSUE TITLE
GPS Data not showing

### ISSUE THREAD

**Issue by bolish** [gpetersen/MMM-BackgroundSlideshowInfo#6]
Hi @gpetersen ,

I don't manage to get GPS data showing in MM...
Would you be so kind and help me?

My config ':

`{

			// 29
			disabled: false,
			module: "MMM-BackgroundSlideshowInfo",
			header: "Picture Info",
			position: 'lower_third',
			config: {
				ignoreFolderNames: ['MagicMirror', 'modules', 'MMM-ImageSlideshow', 'SynoPhoto5Stars'],
				geonameUser: "myusernamewithoutthepassword",
				showGPS: true,
				showGPSFields: ['name','countryCode','adminName1'],
				showCustom: ['exif.FNumber'],
				showFolderNames: true,
				showFileName: false,
				showModel: false,
				
			}
  		},`

Also, terminal is showing the following... which means GPS datas from exif are recognized...

`[BACKGROUNDSLIDESHOWINFO] Getting Geonames for lat/lng: 49.256195/4.028688333333333
`

All others datas are showing fine..

Strange, isn't it?

Thanks already,

**Comment by rts58** [comment]
I think you are missing imagePaths setting.
`imagePaths: ['modules/MMM-BackgroundSlideshow/exampleImages'],`

**Comment by rts58** [comment]
Hi,
Deleted my original reply which was wrong.  Maybe try to delete last comma on config setting showModel line. 

[edit] - that really shouldn't matter either.

**Comment by rts58** [comment]
on mine showGPSFields is showGPSfields

**Comment by bolish** [comment]
Thanks for help @rts58 , I've tried both your options, nothing seems to work for now...Strange...
Can you send your config file extract so I can try with a copy / paste and see what happens?

**Comment by rts58** [comment]
I thought for sure it was because you had the F in uppercase on showGPSfields.  Here is my config extract, hope it helps.
```
		{
			module: 'MMM-BackgroundSlideshowInfo',
			header: "Picture Info",
			position: 'bottom_bar',
			config: {
				ignoreFolderNames: ['modules','pictures','MMM-BackgroundSlideshow'],
				geonameUser: "mygeonameuser",
				//showCustom: ['image.Model','exif.ISO','exif.DateTimeOriginal'],
				showFolderNames: true,
				showFileName: true,
				showGPS: true,
				showGPSfields: ['name','countryCode','adminName1']
			}
		},
```

**Comment by bolish** [comment]
still not.... strange...everything works, jut not the GPS thing...
The geonameUser is the name you have on the top right of the geoname website when logged in correct??
This thing will turn me crazy!

**Comment by rts58** [comment]
Yes, same user.  I copied your config above into mine, and it worked with my userid.  I then changed the userid and it didn't throw any errors but I did not see the GPS data.  So my guess would be it has something to do with your userid.  Did you follow the link in your geonames email to activate your account?

**Comment by bolish** [comment]
I just created a second user name, made sure I activated it through the email link...
Still don't work.

But I finally found the issue!!
Into geonames account (when you click on your account name), there is an hyperlink at the bottom of the page to activate the "webservices". Just clicked on it, works fine since.

It's maybe something new they implemented lately?

Anyway, thank you very much for your support for the analysis step!
